mopro-ffi/src/app_config/ios.rs

```diff
@@ -88,12 +86,23 @@ pub fn build() {
         lib_out
     };
 
-    write_bindings_swift(&swift_bindings_dir);
+    generate_bindings(
+        (manifest_dir.clone() + "/src/mopro.udl").as_str().into(),
```

Comment:
Does not need to `clone` in here.

Reply:
Done, thanks for noticing, it was a leftover :)